### ISSUE TITLE
How to avoid a wait for domain at webview2 startup?

### ISSUE THREAD

**Issue by PaavoHelde** [MicrosoftEdge/WebView2Feedback#1828]
We are developing a desktop app containing some webview2 views. These are used mostly for serving content from the same app, i.e. localhost. Nevertheless, when starting up first time it apparently wants to contact Windows domain or something in there as there is a wait for ca 20 seconds before it starts up, if the domain is not reachable. When starting inside corporate intranet or VPN, there is no such wait. Also, when starting extra instances of the app when there are any msedgewebview2.exe processes already running, there is no wait.

Our company attempts to "control" Edge pretty thoroughly, when running standalone Edge I cannot change the home page locations for example, etc. When starting standalone Edge outside of corporate intranet there is a similar wait before the corporate home page is displayed, but this can be skipped by navigating to another (non-corporate) web page, which loads immediately.

In short, is there a way to configure the webview2 control to skip any network access at startup?

**Comment by champnic** [comment]
Hey @PaavoHelde - There isn't really a "homepage" for WebView2. Have you confirmed the issue is due to a blocked network request? Or is that a guess based on the symptoms/observations? For example, it might be delayed due to anti-virus software on your machine.

How are you initializing the WebView2? You could try having the first navigation go to "about:blank" or something similar.
